app/src/main/res/layout/fragment_my_course.xml

```diff
@@ -81,7 +67,7 @@
 
                 <TextView
                     android:id="@+id/tv_selected"
-                    android:layout_width="wrap_content"
+                    android:layout_width="838dp"
```

Comment:
Don't use fixed size. Not working in smaller device
![Screenshot_1596034021](https://user-images.githubusercontent.com/9145217/88815070-a1738680-d1da-11ea-9e2a-3e4397e3cc16.png)

Reply:
Change fixed width android:layout_width="838dp"